src/krux/pages/file_manager.py

```diff
@@ -97,7 +97,10 @@ def select_file(
                         or SDHandler.dir_exists(path + "/" + filename)
```

Comment:
Hi @jdlcdl ! Here at line 97 we check if it is a directory using `SDHandler.dir_exists(path + "/" + filename)`, then below on line 101 you made the check using `if filename in directories`. Can we create a variable `isDirectory` to reuse?

Reply:
Maybe it's best to do the os.stat directory check higher-up (if it's not a file), and then set isDirectory via "in directories" instead of doing another os.stat for filenames we already know are files?
Or maybe we can avoid an extra os_stat call, since it's in our os.listdir() output, can we assume it's a directory if it's not a file?

Reply:
I don't know which is faster, I tend to think that `SDHandler.dir_exists(path + "/" + filename)` is faster than an `in list` check, because the list check is O(n), but the function dir_exists appears to be O(1). But I'm not worried about performance in this case, just code simplification and reuse.